consensus/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aleph-bft"
-version = "0.36.1"
+version = "0.36.2"
```

Comment:
I suppose you made sure that no public interface has changed? :stuck_out_tongue:

Reply:
I'm pretty confident, this separation between API and internals is already done relatively well in ABFT. :)